extensions/Worker.Extensions.Http/src/HttpResponseAttribute.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker
+{
+    /// <summary>
+    /// Attribute used to mark an HTTP Response on an HTTP Trigger function with multiple output bindings.
+    /// </summary>
+    public sealed class HttpResponseOutputAttribute : OutputBindingAttribute
```

Comment:
Would appreciate name suggestions - I am avoiding `HttpResponseAttribute` since it can be shortened to `HttpResponse` (and VS suggests that), but the source-generator pulls the `HttpResponse` symbol for comparison instead of the attribute in the source-generator in that case.

Reply:
Existing convention for blobs, etc. would just leave it as `HttpOutputAttribute`, right?

Reply:
@mattchenderson I also like that but had a brief convo with @fabiocav where we were between response/output and wondered if response made more sense intuitively ...

Reply:
I'm fine with `HttpResponseOutputAttribute`